Security-Team.md

```diff
@@ -88,6 +89,7 @@ the Node.js program on HackerOne.
 * [@mmarchini](https://github.com/mmarchini) - mary marchini
 * [@MylesBorins](https://github.com/MylesBorins) - Myles Borins
 * [@RaisinTen](https://github.com/RaisinTen) - Darshan Sen
+* [@RafaelGSS](https://github.com/RafaelGSS) - Rafael Gonzaga
```

Comment:
Whoops, this is out of order and should have gone before `@RaisinTen`. But that's OK, it should be fixed by #1217 and is a trivial issue regardless.